src/Compilers/CSharp/Test/Symbol/Symbols/ReplaceOriginalTests.cs

```diff
@@ -14,6 +14,16 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
 {
     public class ReplaceOriginalTests : CSharpTestBase
```

Comment:
@AlekseyTs agree. This is now a pre-req before merging to future so let's do this now.